### PR TITLE
Malee dev

### PR DESCRIPTION
**Pull request by MSeechan**
I created a servlet that takes user input to create a new user entity. The User class is the template that creates new members.  I will need to move this code once I create a branch off Peter's branch so it can probably interact with the account creation UI.

### CODE REVIEW

File: project/src/main/java/com/google/sps/data/User.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.google.sps.data;
+
+public final class User {
```
Comment (SumanjitG) [inline project/src/main/java/com/google/sps/data/User.java:3]: Great User class! All of the parameters/naming clearly indicate what values are stored and used.

File: project/src/main/java/com/google/sps/data/User.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    private final long id;
+    private final String userEmail;
+    private final String userName; 
+    private final String userPassword;
+  
+    //public User(long id, String userName, String userEmail, String userPassword)
+     public User(long id, String userName, String userEmail, String userPassword) {
```
Comment (SumanjitG) [inline project/src/main/java/com/google/sps/data/User.java:9]: We can use this ID to link the user to predefined workout routines?

File: project/src/main/java/com/google/sps/servlets/CreateAccountServlet.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            .set("Email Address", userEmail)
+            .set("Username", userName)
+            .set("password", userPassword)
+            .build();
+        datastore.put(userEntity);
+        
+        // Print values in server logs for testing
+        System.out.println("Email: " + userEmail + "\nUsername: " + userName + "\nPassword: " + userPassword);
+
+        response.sendRedirect("/index.html");  //not sure where to redirect yet
```
Comment (SumanjitG) [inline project/src/main/java/com/google/sps/servlets/CreateAccountServlet.java:50]: Once the user creates the account, they should probably be redirected to their new dashboard? We would probably also need to pass along the user ID like we discussed last week

File: project/src/main/java/com/google/sps/servlets/CreateAccountServlet.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+import javax.servlet.http.HttpServletResponse;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+@WebServlet("/Create-Account")
+public class CreateAccountServlet<User> extends HttpServlet {
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        // get parameters/user input for account creation
```
Comment (SumanjitG) [inline project/src/main/java/com/google/sps/servlets/CreateAccountServlet.java:21]: This function is organized super well and the section separation with comments clearly shows what's happening with the information.

**Review by SumanjitG — APPROVED** [review]
Looks good! I left a few comments on the files.

File: project/src/main/java/com/google/sps/data/User.java (hunk 2 above)
Comment (MSeechan) [reply project/src/main/java/com/google/sps/data/User.java:9]: Yes, this unique ID will be used for that purpose.

**Pull request merged**